RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -24,7 +26,9 @@
     "drop recoPFClusters_*_*_*",
     "keep recoElectronSeeds_*_*_*",
     "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
+    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
+    'keep recoPhotons_gedPhotonsTmp_*_*',
```

Comment:
@slava77 "Intentional" is probably the wrong word, but up until now the PF in the heavyIon scenario only includes the sequences with 'Tmp' appended at the end.  At the time this was set up, I understood that the rest of the PF sequence was to remove PU, which won't do anything in for the HI reco, as we only reconstruct a single PV.